package.json

```diff
@@ -30,6 +30,7 @@
     "webpack-dev-server": "^4.15.1"
```

Comment:
- [ ] It would be be great if you fix your lighthouse error because linters tells you if there are some errors in your app which can be really troublesome in the future 😨

Reply:
I tried but couldn't fix the error, Kindly guide me, on what is wrong in the configuration files.

Reply:
@HAMMAS-SALEEM, Kindly look into the matter and guise me for correction. Thanks